### ISSUE TITLE
translation:sync_and_purge dry run

### ISSUE THREAD

**Issue by nicdal** [aurels/translation-gem#20]
A small request for a --dry-run feature on sync_and_purge dry method.
Could be nice to get a list of which keys it wants to purge before actually doing it.

**Comment by ChillyFlashER** [comment]
This is something I would like too, just this is a server feature so it's not something I can help with.

**Comment by MichaelHoste** [comment]
We just committed that feature on the master branch: https://github.com/aurels/translation-gem#sync-and-show-purgeable

Each time you sync (with `rake translation:sync`), you'll now see something like:

    4 keys/strings are in Translation.io but not in your current branch.
    Execute "rake translation:sync_and_show_purgeable" to list these keys/strings

And if you execute `rake translation:sync_and_show_purgeable`, you'll get the details: 

```
2 YAML keys are in Translation.io but not in your current branch:

 - [enumerize.user.social_status] "Expert"
 - [widgets.interpreter] "Sworn interpreters"

----------
2 GetText strings are in Translation.io but not in your current branch::

 - "Remember me"
 - "You are not logged in"

----------
If you know what you are doing, you can remove them using "rake translation:sync_and_purge"
```

If you execute`rake translation:sync_and_purge` you'll also get the details.

Does that seem to be the expected behaviour for you?

If that seems ok, we'll bump the gem version next week.

**Comment by nicdal** [comment]
Perfect, thank you!

**Comment by MichaelHoste** [comment]
This feature is included in the version [1.10](https://github.com/aurels/translation-gem/releases/tag/v1.10) of the Gem.

Thanks for the feedback!
